sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/GetBoundingBoxesAsync.java

```diff
@@ -50,52 +50,55 @@ public static void main(String[] args) {
-                System.out.printf("Field %s has value %s based on %s with a confidence score "
-                        + "of %.2f.%n",
-                    fieldText, fieldValue.getFieldValue(), fieldValue.getValueText().getText(),
-                    fieldValue.getConfidence());
-            });
+        recognizeFormResult.subscribe(recognizedForms -> {
+            for (int i = 0; i < recognizedForms.size(); i++) {
+                final RecognizedForm recognizedForm = recognizedForms.get(i);
+                System.out.printf("Form %s has type: %s%n", i, recognizedForm.getFormType());
+                // each field is of type FormField
+                //     The value of the field can also be a FormField, or a list of FormFields
+                //     In our sample, it is not.
+                recognizedForm.getFields().forEach((fieldText, fieldValue) -> {
+                    System.out.printf("Field %s has value %s based on %s with a confidence score "
+                            + "of %.2f.%n",
+                        fieldText, fieldValue.getFieldValue(), fieldValue.getValueText().getText(),
+                        fieldValue.getConfidence());
+                });
 
-            // Page Information
-            recognizedForm.getPages().forEach(formPage -> {
-                System.out.printf("-------Recognizing Page %s of Form -------%n", 1);
-                System.out.printf("Has width %s , angle %s, height %s %n", formPage.getWidth(),
-                    formPage.getTextAngle(), formPage.getHeight());
-                // Table information
-                System.out.println("Recognized Tables: ");
-                formPage.getTables().forEach(formTable -> {
-                    formTable.getCells().forEach(formTableCell -> {
-                        System.out.printf("Cell text %s has following words: %n", formTableCell.getText());
-                        // text_content only exists if you set include_text_content to True in your
-                        // function call to recognize_custom_forms
-                        // It is also a list of FormWords and FormLines, but in this example, we only deal with
-                        // FormWords
-                        formTableCell.getElements().forEach(formContent -> {
-                            if (formContent.getTextContentType().equals(TextContentType.WORD)) {
-                                FormWord formWordElement = (FormWord) (formContent);
-                                final StringBuilder boundingBoxStr = new StringBuilder();
-                                if (formWordElement.getBoundingBox() != null) {
-                                    formWordElement.getBoundingBox().getPoints().forEach(point -> {
-                                        boundingBoxStr.append(String.format("[%.2f, %.2f]", point.getX(),
-                                            point.getY()));
-                                    });
+                // Page Information
```

Comment:
Would it also be best to change the forEach for formPage, formTable, formTableCell etc into a for loop?

Reply:
Updated formPage, formTable to loop over an indexed loop but for others the index isn't used in the sample so left as is.